src/NuGet.Core/NuGet.Packaging/PackagePathResolver.cs

```diff
@@ -25,6 +25,13 @@ public PackagePathResolver(string rootDirectory, bool useSideBySidePaths = true)
                     string.Format(Strings.StringCannotBeNullOrEmpty, nameof(rootDirectory)),
                     nameof(rootDirectory));
             }
+            if (!Path.IsPathRooted(rootDirectory))
+            {
+                throw new ArgumentException(
```

Comment:
The tests are failing on mono with `\n'rootDirectory' must contain an absolute path ''.\nParameter name: rootDirectory\n\nExpected: True\nActual: False`

Reply:
@nkolev92 which test. I don't have access to CI.

Reply:
Sorry about that, it's a limitation of our current Azure DevOps setup. 

An example test is: 

`NuGet.CommandLine.Test.NuGetRestoreCommandTest.RestoreCommand_FromHttpSource`

There are 17 failures with I'm pretty sure the same root cause. 

`NuGet.CommandLine.Test.NuGetRestoreCommandTest.RestoreCommand_FromPackagesConfigFile` is another one.

Reply:
Searched for usages of `PackagePathResolver` in code and ensured path was rooted.